src/Controls/src/Xaml/MarkupExtensions/OnPlatformExtension.cs

```diff
@@ -137,16 +139,23 @@ bool TryGetValueForPlatform(out object value)
 				value = Tizen;
 				return true;
 			}
-			if (Device.RuntimePlatform == Device.UWP && UWP != s_notset)
+#pragma warning disable CS0618 // Type or member is obsolete
+			if (Device.RuntimePlatform == Device.WinUI && UWP != s_notset)
```

Comment:
Should we flip the order here and make WinUI return first and the WinUI value in case both UWP and WinUI were specified?